data/json/recipes/other/materials.json

```diff
@@ -126,6 +126,23 @@
       ]
     ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "fake",
+    "result": "polycarbonate_sheet",
```

Comment:
I've tried to look up DIY polycarbonate sheets and everything google is bringing me is that you need to buy a polycarbonate sheet extruding machine.   Plus side if we can't find a way to realistically replace them I just increased their spawns several hundred times a couple days ago.  They probably don't need to be homemade anymore.